LoopKitUI/View Controllers/DismissableHostingController.swift

```diff
@@ -8,8 +8,11 @@
 
 import SwiftUI
 
-
 public class DismissibleHostingController: UIHostingController<AnyView> {
+    public static var cobTintColor: Color = .green
```

Comment:
These should be instance based, not static. It's fine at the app level to assume that all "carb" colors are the same, but not in LoopKit.

Reply:
One additional think I just noticed: the name shouldn't be "COB", which is "Carbs on board", but just "carbohydrate" or "carb" color. I have a slight preference for the full name, rather than the truncated version, but am ok with either.

Reply:
I also prefer carbColor.

Reply:
I was saying I preferred the full name, carbohydrateTintColor, but carb is fine too.

Reply:
I generally don't mind sacrificing a few more characters to make things more understandable for someone new to the project. Typing is really never the bottleneck with programming. :)

Reply:
Sorry, I thought you were suggesting either `carbohydrate` or `carb` were good replacements for `COB` and I chose my preference as a replacement for `COB`. Given `carb` is used `> 3000` times and `carbohydrate` only `195` (often used to display to the user) it would be more consistent to use `carb`. Should I really replace all the instances of `carb` with `carbohydrate`?

Reply:
I think carb is fine. Using 'carbohydrate' is more an opinion than a recommendation. Over time I've definitely shifted more towards avoiding abbreviations, but still find myself using them without thinking.